doc/nrf/app_dev/device_guides/nrf54h/ug_nrf54h20_suit_customize_dfu.rst

```diff
@@ -47,10 +47,11 @@ Software requirements
 
 This guide requires the following software:
 
-* Toolchain Manager - For installing the full |NCS| toolchain.
+* The full |NCS| toolchain.
+  See :ref:`ug_nrf54h20_gs` for instructions on how to install the toolchain.
 * Microsoft's |VSC| - The recommended IDE for |NCS|.
 * |nRFVSC| - An add-on for |VSC| for developing |NCS| applications.
-* nRF Command Line Tools - Mandatory tools for working with |NCS|.
+* nRF Util.
```

Comment:
@maciejpietras @tomchy Could you check this edit? We are deprecating both CLI tools and toolchain manager in 3.0

Reply:
That's fine - we use only the nRF util.